src/difficulty.c

```diff
@@ -11,6 +12,17 @@ u32 GetCurrentDifficultyLevel(void)
     return VarGet(B_VAR_DIFFICULTY);
 }
 
+void SetCurrentDifficulty(u32 desiredDifficulty)
```

Comment:
I wonder if this should be `SetCurrentDifficultyLevel` to match `GetCurrentDifficultyLevel`? (Or vice-versa)